### PR TITLE
Editorial: master is now ES2021

### PR DESCRIPTION
**Pull request by leobalter**

### CODE REVIEW

**Comment by leobalter** [comment]
This PR can only land after we have the final cut for ES2020 approved and sent as the release candidate

**Comment by leobalter** [comment]
Please, observe https://github.com/tc39/ecma402/commits/es2019 for the es2020 cut.

**Comment by leobalter** [comment]
needs updates from #418

**Comment by leobalter** [comment]
rebased

File: spec/introduction.html [hunk 1]
```diff
@@ -38,4 +38,8 @@ <h1>Introduction</h1>
     Leo Balter, Valerie Young, Isaac Durazo<br>
     ECMA-402, 7<sup>th</sup> Edition Project Editors
   </p>
+  <p>
+    Leo Balter<br>
+    ECMA-402, 8<sup>th</sup> Edition Project Editor
+  </p>
```
Comment (leobalter) [inline spec/introduction.html]: cc @spectranaut @isaacdurazo, please let me know if you want to continue the editorship for the 8th Edition (2021) work. I'd be happy to change this.

File: spec/datetimeformat.html [hunk 2]
```diff
@@ -264,7 +264,7 @@ <h1>DateTime Format Functions</h1>
       <h1>PartitionDateTimePattern ( _dateTimeFormat_, _x_ )</h1>
 
       <p>
-        The PartitionDateTimePattern abstract operation is called with arguments _dateTimeFormat_ (which must be an object initialized as a DateTimeFormat) and _x_ (which must be a Number value), interprets _x_ as a time value as specified in ES2015, <emu-xref href="#sec-time-values-and-time-range"></emu-xref>, and creates the corresponding parts according to the effective locale and the formatting options of _dateTimeFormat_. The following steps are taken:
```
Comment (ljharb) [inline spec/datetimeformat.html]: wasn't this originally specified in ES2015 tho? if the date's just going to be updated, why not call it "ECMA 262" and ignore the year?

Comment (leobalter) [reply spec/datetimeformat.html]: the text references to a current value specified. Each new edition invalidates the previous editions so current/parallel edition of ECMAScript has the best accuracy for references.

I agree with renaming it to ECMA-262 or ECMAScript but I'd rather discuss in a separate issue/pr. This PR should be simple enough to just land. If this bit is problematic I'm ok dropping this specific change for now.

Comment (leobalter) [reply spec/datetimeformat.html]: I had to rebase the PR and this was actually meant to be ES2021 here in this branch.

**Pull request merged**